### ISSUE TITLE
TypeError: allureJsCommons.AllureConfig is not a constructor

### ISSUE THREAD

**Issue by vinceis1337** [allure-framework/allure-js-commons#48]
Version `2.0.0.beta-11` has a backwards incompatibility issue because it removes these lines from index.js:
```
var AllureConfig_1 = require("./src/AllureConfig");
Object.defineProperty(exports, "AllureConfig", { enumerable: true, get: function () { return AllureConfig_1.AllureConfig; } });
```

This causes this issue: https://github.com/isaaceindhoven/testcafe-reporter-allure/issues/102 because it is a dependency for that framework.

**Comment by mrtnvh** [comment]
Hi @vinceis1337, I think this project is deprecated and moved to https://github.com/allure-framework/allure-js

There I found the PR that stated the removal of the `AllureConfig` class.
https://github.com/allure-framework/allure-js/pull/323/files

The `AllureConfig` was marked as deprecated for a while, but we failed to notice this. 
So [we](https://github.com/isaaceindhoven/testcafe-reporter-allure) as dependant of the `allure-js-commons` are IMO obliged to follow. I created a [new issue](https://github.com/isaaceindhoven/testcafe-reporter-allure/issues/119) on our side to follow up on this.

I think this issue can be closed.

Thank you for your help!
